### DISCUSSION TITLE
Github desktop doubt

### DISCUSSION THREAD

**Discussion by xignotic** [community/community#126021] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, i m new to github desktop, i have downloaded it but cannot spot a way to co author with someone, how can I do that?

**Comment by m4xy07** [comment, marked as answer]
In the GitHub Desktop App, bottom left corner of the commit desciption box is an icon to add co authors. 

![image](https://github.com/community/community/assets/39182106/01950c50-4a9a-4d6f-9a5e-1cf1854ab2f7)

**Reply by xignotic** [reply]
oh its so small i didnt spot thanks!!!
